pull_into_place/big_jobs/pip_validate.py

```diff
@@ -9,18 +9,30 @@
 import os, sys, subprocess
 from pull_into_place import big_jobs
 
-workspace, job_id, task_id, parameters = big_jobs.initiate()
+workspace, job_id, task_id, parameters, validate_directory = big_jobs.initiate()
 
 designs = parameters['inputs']
 design = designs[task_id % len(designs)]
 test_run = parameters.get('test_run', False)
 
+design_number = '{0:03d}'.format(task_id)
+if len(workspace.loops_path) > 1:
+    fragments_dir = '../' + workspace_validate_path + '/fragments/' + design_number + 'A/'
```

Comment:
Should be changed so that if there's more than one loop, it chooses the biggest one, instead of the first.